XamlControlsGallery/ControlPages/PipsPagerPage.xaml.cs

```diff
@@ -0,0 +1,90 @@
+using System.Collections.Generic;
+using Microsoft.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls;
+
+namespace AppUIBasics.ControlPages
+{
+    public sealed partial class PipsPagerPage : Page
+    {
+        public List<string> Pictures = new List<string>()
+        {
+            "Assets/SampleMedia/LandscapeImage1.jpg",
+            "Assets/SampleMedia/LandscapeImage2.jpg",
+            "Assets/SampleMedia/LandscapeImage3.jpg",
+            "Assets/SampleMedia/LandscapeImage4.jpg",
+            "Assets/SampleMedia/LandscapeImage5.jpg",
+            "Assets/SampleMedia/LandscapeImage6.jpg",
+            "Assets/SampleMedia/LandscapeImage7.jpg",
+            "Assets/SampleMedia/LandscapeImage8.jpg",
+        };
+        public PipsPagerPage()
+        {
+            this.InitializeComponent();
+        }
+
+        private void OrientationComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            if (TestPipsPager1 == null) return;
```

Comment:
Is this check necessary?

Reply:
Likely not, since this event should only trigger when the PipsPager is valid as otherwise the selection wouldn't change. Good point.